### ISSUE TITLE
Naming for NuclearNorm

### ISSUE THREAD

**Issue by adler-j** [odlgroup/odl#1262]
Basically "NuclearNorm" as currently written has some naming issues:

See doc: https://odlgroup.github.io/odl/generated/odl.solvers.functional.default_functionals.NuclearNorm.html

* First, the current definition is actually something like "pointwise nuclear norm and global summation" or something like that. Especially with the new and sexy tensors, we should fix this.
* Further, the code is more global than it markets itself. We could go for the more correct name: https://en.wikipedia.org/wiki/Schatten_norm

**Comment by kohr-h** [comment]
Right, `NuclearNorm` seems to be associated with inner exponent 1, Schatten norm would be the right generalization. As for the integrated part, I don't know. Couldn't that be called Group Schatten norm, like for the standard p-norms?
